ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vxlan_interface.schema.yml

```
@@ -0,0 +1,113 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  vxlan_interface:
+    type: dict
+    keys:
+      Vxlan1:
+        type: dict
+        keys:
+          description:
+            type: str
+          vxlan:
+            type: dict
+            keys:
+              source_interface:
+                type: str
+                display_name: Source Interface Name
+              mlag_source_interface:
+                type: str
+              udp_port:
+                type: int
+                convert_types:
+                - str
+              virtual_router_encapsulation_mac_address:
+                type: str
+                description: |
+                  < mlag-system-id | ethernet_address (H.H.H) >
```

Comments can't include `|`this causes the rendered markdown table to think this is another column
```suggestion
                  mlag-system-id or ethernet_address (H.H.H)
```